go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/moby/buildkit
 
-go 1.21.0
+go 1.22.0
```

Comment:
can just be `1.22`

Reply:
Thanks! `1.22` makes more sense to me, but since the same file was using `1.21.0` I kept it pinned to a patch release just in case there was some BuiltKit convention I was not aware of. I've updated it to use `1.22`.

Reply:
> but since the same file was using `1.21.0`

this was because of a peer dep forcing it, see https://github.com/moby/buildkit/pull/5118#discussion_r1664266928

Reply:
ah actually I was wrong, exp/trace also set patch so we need it as well https://cs.opensource.google/go/x/exp/+/master:go.mod;l=3

Reply:
I see, good point. I've changed it back to 1.22.0.